### PR TITLE
Feature curriculum content

### PR DESCRIPTION
**Pull request by VHDLNER**
i create this branch to resolve the  issue with feature order-content

### CODE REVIEW

File: src/main/java/com/revature/controllers/ContentController.java [hunk 1]
```diff
@@ -92,7 +92,11 @@
 	public ResponseEntity<Set<Link>> getLinksByContentId(@PathVariable int id) {
 		return ResponseEntity.ok(contentService.getLinksByContentId(id));
 	}
-
+	// return list of links attached to a given curriculum
+	@GetMapping("/curricula/{id}/links")
```
Comment (carolynrehm) [inline src/main/java/com/revature/controllers/ContentController.java]: This would come out to be /content/curricula/{id}/links. We shouldn't need /content/

Comment (carolynrehm) [reply src/main/java/com/revature/controllers/ContentController.java]: perhaps move this method to the curriculum controller?

Comment (VHDLNER) [reply src/main/java/com/revature/controllers/ContentController.java]: the reason i put it on content is that the front end used this method to extract content from links

Comment (VHDLNER) [reply src/main/java/com/revature/controllers/ContentController.java]: if i moved it to curriculum controller  the mapping will be : 
/content/{id}

Comment (carolynrehm) [reply src/main/java/com/revature/controllers/ContentController.java]: Why would content be in it at all? The class is annotated with @RequestMapping("/curricula") and you could have a method which has a @GetMapping with "/{id}/links"

File: src/test/java/com/revature/services/CurriculumServiceTest.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 import org.testng.annotations.Test;
 import com.revature.cmsforce.CMSforceApplication;
 import com.revature.entities.Curriculum;
+import com.revature.entities.CurriculumModule;
```
Comment (carolynrehm) [inline src/test/java/com/revature/services/CurriculumServiceTest.java]: why is this needed?

File: src/main/java/com/revature/controllers/ContentController.java (hunk 1 above)
Comment (VHDLNER) [reply src/main/java/com/revature/controllers/ContentController.java]: you right it useless to include content, i'll move it to curriculum controller

File: src/test/java/com/revature/services/CurriculumServiceTest.java (hunk 2 above)
Comment (VHDLNER) [reply src/test/java/com/revature/services/CurriculumServiceTest.java]: i didn't work on this test, I don't know how it got added

File: src/test/java/com/revature/services/CurriculumServiceTest.java [hunk 3]
```diff
@@ -25,6 +25,7 @@
 	@Mock
 	private CurriculumRepository crMock;
 	private Curriculum curriculum;
+
```
Comment (carolynrehm) [inline src/test/java/com/revature/services/CurriculumServiceTest.java]: Can we take this extra line out? We shouldn't need CurriculumServiceTest in this PR

**Comment by VHDLNER** [comment]
i moved the method to the curriculum controller

**Review by carolynrehm — CHANGES_REQUESTED** [review]
These files look good:
src/main/java/com/revature/controllers/CurriculumController.java
src/main/java/com/revature/services/CurriculumService.java
src/main/java/com/revature/services/CurriculumServiceImpl.java
src/test/java/com/revature/controllers/ContentControllerTest.java
src/test/java/com/revature/controllers/CurriculumControllerTest.java

These files seem unnecessarily included:
src/main/java/com/revature/controllers/ContentController.java
src/main/java/com/revature/services/ContentService.java
src/main/java/com/revature/services/ContentServiceImpl.java
src/test/java/com/revature/services/CurriculumServiceTest.java

Please remove those files from the pull request (make sure they match dev) and then we can merge this in @VHDLNER

**Review by carolynrehm — CHANGES_REQUESTED** [review]
Still need to remove:

src/main/java/com/revature/services/ContentService.java
src/main/java/com/revature/services/ContentServiceImpl.java
src/test/java/com/revature/services/CurriculumServiceTest.java

**Review by carolynrehm — APPROVED** [review]
